components/authz-service/server/v2/system.go

```diff
@@ -60,6 +60,12 @@ func SystemPolicies() []*storage.Policy {
 				Resources: []string{"iam:policyVersion"},
 				Projects:  []string{constants.AllProjectsID},
 			},
+			{
+				Effect:    storage.Allow,
+				Actions:   []string{"iam:rules:status"},
+				Resources: []string{"iam:rules"},
+				Projects:  []string{constants.AllProjectsID},
```

Comment:
🤔 We can have more than one actions/resource per statement in IAMv2, why not make use of that?

Reply:
While that is true, it is not the same thing, is it? With `Actions: {a, b}` and `Resources: {1, 2}` that would allow any combination rather than specifically `a1` and `b2`.

Reply:
Yup, but that distinction doesn't matter with the way the actions are prefixed, does it?

(Regretably, what we have here generates two cases in partial eval; what I have proposed generates four.)